post_office/management/commands/send_queued_mail.py

```diff
@@ -34,6 +34,11 @@ def add_arguments(self, parser):
             type=int,
             help='"0" to log nothing, "1" to only log errors',
         )
+        parser.add_argument(
+            '-x', '--exit-after-batch',
```

Comment:
I think this feature makes sense. Can you slightly change this to `--max-batches`? `--max-batches=2` means it will exit after two batches of emails are sent.